include/linux/tcp.h

```diff
@@ -342,6 +342,7 @@ struct tcp_sock {
 	u32	snd_cwnd_clamp; /* Do not allow snd_cwnd to grow above this */
 	u32	snd_cwnd_used;
 	u32	snd_cwnd_stamp;
+	u32	snd_cwnd_before_idle_restart; /* used by MPTCP ECF scheduler */
```

Comment:
This should probably go into ```mptcp_sched``` of ```struct mptcp_tcp_sock```.

Reply:
Sure, I will just put this in `ecfsched_priv`, where all the scheduling-related subflow information are located.